assets/maugallery.js

```diff
@@ -240,24 +226,18 @@
         return;
       }
       $(".active-tag").removeClass("active active-tag");
-      $(this).addClass("active-tag");
+      $(this).addClass("active active-tag");
```

Comment:
Maybe could we store `active active-tag` into a variable, and use it on this line and on line 209:

```js
var tagItems =
        `<li class="nav-item"><span class="nav-link ${varName}"  data-images-toggle="all">Tous</span></li>`;
```